docs/intelligence_center/integrations.md

```diff
@@ -0,0 +1,29 @@
+# Intelligence Center external integrations
+
+The intelligence center data can be consummed using several third party integrations.
+
+## MISP Feed
+
+The default feed is available as a MISP feed.
+
+It can be added to an existing MISP instance by following [MISP's documentation](https://www.circl.lu/doc/misp/managing-feeds/).
+
+The following field values are required for the feed to work properly:
+
+-   Input Source: Network
+-   URL: `https://api.sekoia.io/api/v2/inthreat/misp`
+-   Source Format: MISP Feed
+-   Headers: `Authorization: Bearer <APIKEY>`
+-   Enabled: `True`
+
+You then need to make sure you have a scheduled task in place to regularly fetch the feed's content.
+
+## OpenCTI connector
+
+An [OpenCTI](https://www.opencti.io) connector is available to consumme a feed.
+All the instruction to run it are available at the connector github repository: https://github.com/SEKOIA-IO/opencti-connector.
+
+## Cortex Analyser
+
+SEKOIA is also providing a [cortex analyzer](https://github.com/TheHive-Project/Cortex-Analyzers/tree/master/analyzers/SEKOIAIntelligenceCenter) to enrich data in [THEHIVE](https://thehive-project.org/) ecosystem.
+To setup the analyzer please follow the following guide: https://github.com/TheHive-Project/CortexDocs/blob/master/analyzer_requirements.md.
```

Comment:
```suggestion
To setup the analyzer please [follow the following guide](https://github.com/TheHive-Project/CortexDocs/blob/master/analyzer_requirements.md).
```

Reply:
To setup the analyzer please [follow this guide](https://github.com/TheHive-Project/CortexDocs/blob/master/analyzer_requirements.md).